.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseTimeZoneParser.cs

```diff
@@ -69,6 +69,12 @@ public static string ConvertMinsToRegularFormat(int offsetMins)
             return TimeSpan.FromMinutes(offsetMins).ToString(@"hh\:mm");
         }
 
+        public static string NormalizeText(string text)
+        {
+            text = Regex.Replace(text, @"\s+", " ");
+            return Regex.Replace(text, @"timezone", "time");
```

Comment:
Why this new timezone->time normalization? Wouldn't it be better to actually just remove either word from the end of the match, trim, and then search in the dictionary?